src/EMS/RingdownStatus.js

```diff
@@ -19,23 +20,58 @@ function RingdownStatus({ className, onStatusChange, ringdown }) {
     onStatusChange(ringdown, Ringdown.Status.REDIRECTED);
   }
 
+  useEffect(() => {
+    switch (ringdown.payload.patientDelivery.currentDeliveryStatus) {
+      case 'RECEIVED':
+        setRingdownStatus('Delivered');
+        break;
+      case 'ARRIVED':
+        setRingdownStatus('Confirmed');
+        break;
+      case 'OFFLOADED':
+        setRingdownStatus('Pending');
+        break;
+      default:
+        break;
+    }
+  }, [ringdown.payload.patientDelivery.currentDeliveryStatus]);
+
   return (
     <div className={classNames('usa-accordion', className)}>
       <div className="usa-accordion__content">
         <fieldset className="usa-fieldset">
           <h3 className="h1 margin-0">{ringdown.hospital.name}</h3>
-          <h4 className="text-base-light margin-top-2">ETA {ringdown.etaMinutes} minutes</h4>
+          <h4 className="text-base-light margin-bottom-0">
+            Ringdown Status:{' '}
+            <span className={`ringdown--${ringdown.payload.patientDelivery.currentDeliveryStatus.replace(/\s/g, '')}`}>
+              {ringdownStatus}
+            </span>
+          </h4>
+          <h4 className="text-base-light">
+            ETA:
+            <b style={{ color: 'black' }}>
```

Comment:
It's generally preferrable NOT to use inline styles, but to define appropriate classes and put the styles in the accompanying stylesheet. It reduces clutter in the output markup.